docs/docs/APIGuide/Layers/Recurrent-Layers.md

```diff
@@ -1558,4 +1561,232 @@ array([[[[ 0.1       ,  0.2       ],
         [[ 0.30000001,  0.40000001],
          [ 0.30000001,  0.40000001]]]], dtype=float32)
 ```
+---
+
+## MultiCell ##
+
+**Scala:**
+```scala
+val model = MultiCell(cells = multiCells)
+
+```
+**Python:**
+```python
+model = MultiCell(cells = multiCells)
+```
+
+A cell that stack multiple simple rnn cells. Only works with RecurrentDecoder. If you want to
+stack multiple cells with Recurrent. Use Sequential().add(Recurrent(cell)).add(Recurrent(cell))... instead
+
+Parameters:
+
+* `cells` list of RNNCells that will be composed in this order.
```

Comment:
Can it be a list of LSTM cells? Or a list of cells mixing RNN/LSTM/GRU etc.?

Reply:
It can be a list of different rnn cells, if output of last cell can be can be processed by the following cell.